client/my-sites/site-settings/media-settings.jsx

```diff
@@ -115,14 +115,14 @@ class MediaSettings extends Component {
 						<div className="site-settings__info-link-container">
 							<InfoPopover position="left">
 								<ExternalLink target="_blank" icon href="https://jetpack.com/support/photon" >
-									{ translate( 'Learn more about Photon.' ) }
+									{ translate( 'Learn more' ) }
 								</ExternalLink>
 							</InfoPopover>
 						</div>
 						<JetpackModuleToggle
 							siteId={ siteId }
 							moduleSlug="photon"
-							label={ translate( 'Speed up your images and photos with Photon' ) }
+							label={ translate( 'Speed up images and photos' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Speed up your images and photos with Photon.' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/remove-photon-from-settings-media-card)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).